common/exprParser/helpers/sqlUtils.js

```diff
@@ -0,0 +1,54 @@
+const { types } = require('./types')
+
+const getSqlOperator = op => {
+  switch (op) {
+  case '||':
+    return 'OR'
+  case '&&':
+    return 'AND'
+  case '===':
+    return '='
+  case '!==':
+    return '!='
+  default:
+    throw new Error(`Unknown operator: ${op}`)
+  }
+}
+
+const binaryToString = (node, params) => {
+  const left = toString(node.left, params)
+  const right = toString(node.right, left.params)
+
+  return {
+    str: `${left.str} ${getSqlOperator(node.operator)} ${right.str}`,
```

Comment:
and here use R.prop(node.operator, sqlOperators)

Reply:
Kept it as a function to throw an exception if the operator is unknown